conda_forge_tick/auto_tick.py

```diff
@@ -222,27 +211,8 @@ def _host_run_test_dependencies(meta_yaml: "MetaYamlTypedDict") -> Set["PackageN
     The complicated logic here is mainly to support not including a
     `host` section, and using `build` instead.
     """
-    rq = set()
-    for block in [meta_yaml] + meta_yaml.get("outputs", []) or []:
-        req: "RequirementsTypedDict" = block.get("requirements", {}) or {}
-        # output requirements given as list (e.g. openmotif)
-        if isinstance(req, list):
-            rq.update(_requirement_names(req))
-            continue
-
-        # if there is a host and it has things; use those
-        if req.get("host"):
-            rq.update(_requirement_names(req.get("host", [])))
-        # there is no host; look at build
-        elif req.get("host", "no host") not in [None, []]:
-            rq.update(_requirement_names(req.get("build", []) or []))
-        rq.update(_requirement_names(req.get("run", []) or []))
-
-        # add testing dependencies
-        test: "TestTypedDict" = block.get("test", {})
-        rq.update(_requirement_names(test.get("requirements")))
-        rq.update(_requirement_names(test.get("requires")))
-
+    _ = meta_yaml["requirements"]
+    rq = (_["host"] or _["build"]) | _["run"] | _["test"]
```

Comment:
Whoa. This underscore thing threw me for a loop but I see it now. I wonder if a named var is better?

Reply:
Most likely it will be I was coding this without an IDE, so some code could use some work.